### DISCUSSION TITLE
Cherry-pick from arbitrary commit hash

### DISCUSSION THREAD

**Discussion by NN---** [gitextensions/gitextensions#9986] [General] [answered]
I would like to have in cherry pick command to put any commit hash without the hassle of dialog and etc.

It should be as simple as command: `git cherry-pick myhash`

**Comment by maraf** [comment, marked as answer]
You can create a scripts for it (Tools -> Settings -> Scripts)

![image](https://user-images.githubusercontent.com/10020471/167614730-d790a125-b313-423e-8db4-22e8ef744f49.png)
